oauth2_http/java/com/google/auth/mtls/WorkloadCertificateConfiguration.java

```diff
@@ -0,0 +1,64 @@
+package com.google.auth.mtls;
+
+import com.google.api.client.json.GenericJson;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.JsonObjectParser;
+import com.google.api.client.json.gson.GsonFactory;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Map;
+
+public class WorkloadCertificateConfiguration {
```

Comment:
This class looks like it's just being used from X509Provider class. Can this be package-private scope?